taxi/views.py

```diff
@@ -57,7 +88,29 @@ class ManufacturerDeleteView(LoginRequiredMixin, generic.DeleteView):
 class CarListView(LoginRequiredMixin, generic.ListView):
     model = Car
     paginate_by = 5
-    queryset = Car.objects.all().select_related("manufacturer")
+
+    def get_context_data(self, *, object_list=None, **kwargs):
```

Comment:
add additional typehint for list inner objects like list[ANY_TYPE]
use typehint like: Optional[YOUR_VALUE_TYPE] = None